toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -109,108 +110,142 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
+		sort.Strings(rpmConflicts)
+		for _, conflict := range rpmConflicts {
+			conflictsLogger("--> %s", conflict)
+		}
+	}
+
+	if len(srpmConflicts) != 0 {
+		conflictsLogger("\033[31mSRPM conflicts with toolchain:\033[33m")
+		sort.Strings(srpmConflicts)
+		for _, conflict := range srpmConflicts {
+			conflictsLogger("--> %s", conflict)
 		}
 	}
 
 	if len(failedSRPMs) != 0 {
-		logger.Log.Info("Failed SRPMs:")
-		for _, failure := range failedSRPMs {
+		logger.Log.Info("\033[31mFailed SRPMs:\033[33m")
+		keys := getSortedKeys(failedSRPMs)
+		for _, key := range keys {
+			failure := failedSRPMs[key]
 			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
 		}
 	}
 
 	if len(failedSRPMsTests) != 0 {
-		logger.Log.Info("Failed SRPMs tests:")
-		for _, failure := range failedSRPMsTests {
+		logger.Log.Info("\033[31mFailed SRPMs tests:\033[33m")
+		keys := getSortedKeys(failedSRPMsTests)
+		for _, key := range keys {
+			failure := failedSRPMsTests[key]
 			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
 		}
 	}
 
-	if len(blockedSRPMs) != 0 {
-		logger.Log.Info("Blocked SRPMs:")
-		for srpm := range blockedSRPMs {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+	if len(prebuiltSRPMs) != 0 {
+		logger.Log.Info("\033[32mPrebuilt SRPMs:\033[33m")
+		keys := getSortedKeys(prebuiltSRPMs)
+		for _, prebuiltSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(prebuiltSRPM))
 		}
 	}
 
-	if len(blockedSRPMsTests) != 0 {
-		logger.Log.Info("Blocked SRPMs tests:")
-		for srpm := range blockedSRPMsTests {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+	if len(builtSRPMs) != 0 {
+		logger.Log.Info("\033[32mBuilt SRPMs:\033[33m")
+		keys := getSortedKeys(builtSRPMs)
+		for _, builtSRPM := range keys {
+			logger.Log.Infof("--> %s ", filepath.Base(builtSRPM))
 		}
 	}
 
-	if len(unresolvedDependencies) != 0 {
-		logger.Log.Info("Unresolved dependencies:")
-		for dependency := range unresolvedDependencies {
-			logger.Log.Infof("--> %s", dependency)
+	if len(testedSRPMs) != 0 {
+		logger.Log.Info("\033[32mTested SRPMs:\033[33m")
+		keys := getSortedKeys(testedSRPMs)
+		for _, testedSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(testedSRPM))
 		}
 	}
 
-	if len(rpmConflicts) != 0 {
-		conflictsLogger("RPM conflicts with toolchain: ")
-		for _, conflict := range rpmConflicts {
-			conflictsLogger("--> %s", conflict)
-		}
-	}
+	logger.Log.Info("---------------------------")
+	logger.Log.Info("--------- Summary ---------")
+	logger.Log.Info("---------------------------")
 
-	if len(srpmConflicts) != 0 {
-		conflictsLogger("SRPM conflicts with toolchain: ")
-		for _, conflict := range srpmConflicts {
-			conflictsLogger("--> %s", conflict)
-		}
+	logger.Log.Infof("\033[32mNumber of built SRPMs:             %d\033[33m", len(builtSRPMs))
```

Comment:
We should order these the same way we order the other lists. Maybe we want a dedicated function to print these since we run it twice?